config/idm-endpoints/scripts-content/echo.js

```diff
@@ -0,0 +1,79 @@
+/*
```

Comment:
Is this a test script?

Reply:
yes, this is the default script proving different features of an IDM endpoint. I've included it for now, as it's a good reference for our implementation, will remove it later.

Reply:
Yep, makes sense.